### PR TITLE
Move deployment from Dockerfile setup to host volume setup

### PR DESCRIPTION
**Pull request by flomotlik**
We don't want deployments to use Onbuild and Dockerfiles, thus we're
moving them to use host volumes from now on and remove the onbuild
configuration.

### CODE REVIEW

**Comment by jschley** [comment]
lgtm :+1:

File: codeship-steps.yml [hunk 1]
```diff
@@ -1,10 +1,11 @@
-- name: trigger_build
-  service: awsdeployment
-  command: true
-- service: integrationtest
+- service: awsdeployment
+  command: bash -c " mkdir -p /deploy/tmp && zip -r /deploy/tmp/upload_to_s3.zip /deploy/test/upload_to_s3"
```
Comment (ngauthier) [inline codeship-steps.yml]: please use a script in bin if possible

**Comment by ngauthier** [comment]
minor comment, not blocking. :+1:

File: codeship-steps.yml (hunk 1 above)
Comment (flomotlik) [inline codeship-steps.yml]: Moved it into test/test_zip_available.sh so its clear that its a test

**Pull request merged**